resources/profiles/Elegoo.json

```
@@ -817,6 +953,22 @@
 		{
 			"name": "Elegoo Neptune 4 Pro (0.8 nozzle)",
 			"sub_path": "machine/Elegoo Neptune 4 Pro (0.8 nozzle).json"
+		},
+		{
+			"name": "Elegoo OrangeStorm Giga 4.0 nozzle",
+			"sub_path": "machine/Elegoo OrangeStorm Giga 4.0 nozzle.json"
+		},
+		{
+			"name": "Elegoo OrangeStorm Giga 1.0 nozzle",
+			"sub_path": "machine/Elegoo OrangeStorm Giga 1.0 nozzle.json"
+		},
+		{
+			"name": "Elegoo OrangeStorm Giga 6.0 nozzle",
+			"sub_path": "machine/Elegoo OrangeStorm Giga 6.0 nozzle.json"
+		},
+		{
+			"name": "Elegoo OrangeStorm Giga 8.0 nozzle",
+			"sub_path": "machine/Elegoo OrangeStorm Giga 8.0 nozzle.json"
```

This seems to cause the error, as it's looking for the wrong files.
Classic typo :D 

```suggestion
		},
		{
			"name": "Elegoo OrangeStorm Giga 0.4 nozzle",
			"sub_path": "machine/Elegoo OrangeStorm Giga 0.4 nozzle.json"
		},
		{
			"name": "Elegoo OrangeStorm Giga 1.0 nozzle",
			"sub_path": "machine/Elegoo OrangeStorm Giga 1.0 nozzle.json"
		},
		{
			"name": "Elegoo OrangeStorm Giga 0.6 nozzle",
			"sub_path": "machine/Elegoo OrangeStorm Giga 0.6 nozzle.json"
		},
		{
			"name": "Elegoo OrangeStorm Giga 0.8 nozzle",
			"sub_path": "machine/Elegoo OrangeStorm Giga 0.8 nozzle.json"
```

---

Thank you so much! You have no idea how much time I wasted looking at that file and not finding anything!

---

Guessed as much by your increasingly desperate commit messages :D

---

lol

---

Hi @cochcoder, I apologize for not being available. I was traveling and then fell ill for a while. I'm really grateful that @arough007 could step in to help. Thank you both so much!

---

> Hi @cochcoder, I apologize for not being available. I was traveling and then fell ill for a while. I'm really grateful that @arough007 could step in to help. Thank you both so much!

It's not a issue at all! Thank you for all your work on this slicer and thank you to @arough007 again for finding the source of this error!